core/new-gui/src/app/dashboard/service/workflow-executions/workflow-executions.service.ts

```diff
@@ -27,13 +27,28 @@ export class WorkflowExecutionsService {
     });
   }
 
+  groupSetIsBookmarked(wid: number, eIds: number[], isBookmarked: boolean): Observable<Object> {
+    return this.http.put(`${WORKFLOW_EXECUTIONS_API_BASE_URL}/set_execution_bookmarks`, {
+      wid,
+      eIds,
+      isBookmarked,
+    });
+  }
+
   deleteWorkflowExecutions(wid: number, eId: number): Observable<Object> {
```

Comment:
will this be called? didn't we change the backend API to always accept an array?